.changeset/clever-jokes-chew.md

```diff
@@ -0,0 +1,6 @@
+---
+"@comet/eslint-config": major
+"@comet/eslint-plugin": major
+---
+
+Add eslint rule that enforces absolute (@src/...) import from other modules (src/module)
```

Comment:
I'd suggest changing the changeset to this

```md
Add new ESLint rule to enforce absolute imports when importing from other modules

For instance, an import `import { AThingInModuleA } from "../moduleA/AThingInModuleA"` in module `B` needs to be imported as `import { AThingInModuleA } from "@src/moduleA/AThingInModuleA"`.
The default source root `"./src"` and alias `"@src"` can be changed via the rule's `sourceRoot` and `sourceRootAlias` options.
This rule will be enforced by `@comet/eslint-config` in the next major release.
```

Also, this would then be a minor change for eslint-plugin, and no change for eslint-config